advanced_scripts/screens.rpy

```diff
@@ -1591,7 +1591,7 @@ screen month_input(ok_action):
                 style "confirm_prompt"
                 xalign 0.5
 
-            input default "" value VariableInputValue("month") length 15 allow "ABCDEFGHIJKLMNOPQRSTUVabcdefghijklmnopqrstuv"
+            input default "" value VariableInputValue("month") length 15 allow "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"

```

Comment:
I think we could make this a constant, something like `LETTERS_ONLY`, that way we can re-use it without constantly re-defining the string.

Plus it makes looking at it on a glance much easier.

Reply:
pushed a commit for that